src/Monticello/MCPackageLoader.class.st

```diff
@@ -242,16 +241,19 @@ MCPackageLoader >> warnAboutDependencies [
 ]
 
 { #category : 'private' }
-MCPackageLoader >> warnAboutErrors: errorDefinitions [
+MCPackageLoader >> warnAboutErrors: errorsDict [
 
 	self notify: (String streamContents: [ :s |
 			 s
 				 nextPutAll: 'The following definitions had errors while loading.  Press Proceed to try to load them again (they may work on a second pass):';
 				 cr.
-			 errorDefinitions do: [ :ea |
+			 errorsDict keysDo: [ :ea |
 				 s
 					 space;
 					 space;
 					 nextPutAll: ea summary;
+					 nextPutAll: ' (';
+					 nextPutAll: ((errorsDict at: ea) description asString);
+					 nextPutAll: ')';
```

Comment:
```suggestion
			 errorsDict keysAndValuesDo: [ :definition :error |
				 s
					 space;
					 space;
					 nextPutAll: definition summary;
					 nextPutAll: ' (';
					 nextPutAll: (error description asString);
					 nextPutAll: ')';
```

Reply:
@jecisc may be we should do your changes and integrate this PR

Reply:
I don't have the rights to push my changes to this PR. I'll just integrate it. My changes were for readability only